lib/generators/gql/templates/input_type.rb

```diff
@@ -1,9 +0,0 @@
-module Types
-  module Input
-    class <%= @model_name %>Input < Types::BaseInputObject
-  <% @fields.each do |field| -%>
-    argument :<%= field[:name] %>, <%= field[:gql_type] %>, required: false
-  <% end %>
-    end
-  end
-end
```

Comment:
What was your thinking here? I find the generator template files a bit easier to reason about than generating files completely in code.

Reply:
Heh, glad you asked. So here's what the code looked like before I moved it: 

```erb
<%= @namespace.each_with_index.map { |name, i| "  " * i + "module #{name}" }.join("\n") %>
<%= "  " * @namespace.size %>class <%= @name %>Type < <%= @superclass %>
  <% @fields.each do |field| -%>
  field :<%= field[:name] %>, <%= field[:gql_type] %>, null: true
  <% end %>
<%= "  " * (@namespace.size + 0) %>end
<%= @namespace.size.downto(0).map { |i| "  " * i  + "end" }.join("\n") %>
```

As a long time and Perl developer, I can say with confidence that this is much more difficult to maintain than what we have now. 

There's also a lot of duplicate functionality across templates. How to expose this as "view helpers"? I don't know. And this is not the type of code that should be copied and pasted to multiple places.

Reply:
Oh and I don't think this code is even complete.

Reply:
@ajsharp thoughts here? I'd like to wrap this up before I forget how it works.

Reply:
@sshaw apologies, i've been slammed. I'm good with all this. Will merge this now. Thanks!

Reply:
No worries, I know how it goes!

But let me make 1 more commit before you merge. Give me < 48 hours, thanks!